gradle/version.properties

```diff
@@ -25,6 +25,9 @@ crate_jdbc=2.5.1
 elasticsearch=6.5.1
```

Comment:
Can we remove this?

Reply:
It's still required for some dependencies (elasticsearch-cli), which we fetch instead of building. We can follow up on this to remove it later on I guess